src/sentry/event_manager.py

```diff
@@ -462,9 +462,6 @@ def save_error_events(
 
         is_reprocessed = is_reprocessed_event(job["data"])
 
-        # This metric can be used to track how many error events there are per platform
-        metrics.incr("save_event.error", tags={"platform": job["event"].platform or "unknown"})
```

Comment:
We now have this metric instead:
https://github.com/getsentry/sentry/blob/c2854dc5ed7ef766b71b263aa70a7b63fdb28933/src/sentry/event_manager.py#L444

Reply:
If they do the same thing, the new one is indeed better, because `save_event.error` makes it sound like there was a problem saving the event, not that what we're doing is successfully saving an error event.